package.json

```diff
@@ -25,6 +28,7 @@
     "dev": "parcel demo/index.html -d .devserver --open",
     "build": "node rollup.build.js && bundlesize",
     "test": "jest --coverage",
+    "test:browser": "parcel test/browser/index.html -d .devserver --open",
```

Comment:
We should probably implement automated visual tests, maybe cypress? I'll take a look at it